### PR TITLE
Stub test case to verify pulp3 logs are captured

### PR DESCRIPTION
**Pull request by shweta83**
Added stub test case to verify Pulp3 Logs are being captured using correlation id.

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
Can one of the admins verify this patch?

File: tests/foreman/cli/test_logging.py [hunk 1]
```diff
@@ -230,3 +230,29 @@ def test_positive_logging_from_dynflow(module_org):
                 assert match, "Request ID not found"
                 logger.info("Request ID found for logging from dynflow ")
     assert POST_line_found, "The POST command to enable a repo was not found in logs."
+
+
+@pytest.mark.tier4
+@pytest.mark.stubbed
+@pytest.mark.skip_if_open("BZ:1732522")
```
Comment (swadeley) [inline tests/foreman/cli/test_logging.py:237]: That bug is VERIFIED so we do not want to spend processor time checking if its open.

File: tests/foreman/cli/test_logging.py [hunk 2]
```diff
@@ -230,3 +230,29 @@ def test_positive_logging_from_dynflow(module_org):
...
+def test_positive_logging_verify_pulp3_logs():
```
Comment (swadeley) [inline tests/foreman/cli/test_logging.py:238]:
```suggestion
def test_positive_logging_from_pulp3():
```
I think to many words with similar meanings

**Review by swadeley — APPROVED** [review]
ACK with suggestions

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**